tests/test_integration.py

```diff
@@ -336,6 +510,19 @@ def elapsed():
     raise RuntimeError("reached max poll time: {e} > {m}".format(e=elapsed(), m=max_poll_time))
 
 
+def assert_job_status(job: BatchJob, expected_status):
+    # If the next assert is going to fail, then first show the logs of the job.
+    if job.status != expected_status:
```

Comment:
```suggestion
    if job.status() != expected_status:
```

Also further in this function, but I would store it in a local var for simplicity

Reply:
Done.